samples/matter/thermostat/src/binding_handler.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "binding_handler.h"
+#include "temp_sensor_manager.h"
+#include "temperature_measurement/sensor.h"
+
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app-common/zap-generated/ids/Commands.h>
+#include <app/CommandSender.h>
+#include <app/OperationalSessionSetup.h>
+#include <app/clusters/bindings/BindingManager.h>
+#include <app/clusters/bindings/bindings.h>
+#include <app/data-model/Decode.h>
+#include <app/util/binding-table.h>
+#include <controller/InvokeInteraction.h>
+#include <controller/ReadInteraction.h>
+#include <lib/support/CodeUtils.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace chip;
+using namespace chip::app;
+
+void BindingHandler::Init()
+{
+	DeviceLayer::PlatformMgr().ScheduleWork(InitInternal);
+}
+
+void BindingHandler::TemperatureMeasurementReadHandler(const EmberBindingTableEntry &binding,
+						       OperationalDeviceProxy *peer_device, void *context)
+{
+	VerifyOrReturn(context != nullptr, LOG_ERR("Invalid context for Thermostat handler"););
+	BindingData *data = static_cast<BindingData *>(context);
+
+	TemperatureMeasurementUnicastBindingRead(data, binding, peer_device);
+}
+
+void BindingHandler::TemperatureMeasurementUnicastBindingRead(BindingData *data, const EmberBindingTableEntry &binding,
+							      OperationalDeviceProxy *peer_device)
+{
+	auto onSuccess = [](const ConcreteDataAttributePath &attributePath, const auto &dataResponse) {
+		ChipLogProgress(NotSpecified, "Read Thermostat attribute succeeds");
+		int16_t ResponseValue = dataResponse.Value();
```

Comment:
```suggestion
		int16_t responseValue = dataResponse.Value();
```
nitpicking the consistent naming scheme for all variables